web-app/src/app/layout.tsx

```diff
@@ -42,14 +41,12 @@ export default async function RootLayout({
           "bg-background min-h-svh antialiased",
         )}
       >
-        <NuqsAdapter>
```

Comment:
Why did you remove the NuqsAdapter? Please readd it. We need it for the agents page in the logged in state.

Reply:
4bbc50ddb302c1ec510728c32931cb3624fff923